Library/Formula/scirius.rb

```diff
@@ -0,0 +1,49 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius"
```

Comment:
Well that's something... I've done all you told me to do and got this:

    $ brew fetch scirius
    ==> Downloading https://github.com/StamusNetworks/scirius/archive/scirius-1.0.tar.gz
    ==> Downloading from https://codeload.github.com/StamusNetworks/scirius/tar.gz/scirius-1.0
    ######################################################################## 100.0%
    Downloaded to: /Library/Caches/Homebrew/scirius-1.0.tar.gz
    SHA1: afa3838f4fd431569684f55e7545c0490021104d
    SHA256: fde16b3ac8f95e2990a44a9d85aa7c3ee6b55e59af9c27ded7319db19c19b2f5
    Warning: Cannot verify integrity of scirius-1.0.tar.gz
    A checksum was not provided for this resource
    For your reference the SHA256 is: fde16b3ac8f95e2990a44a9d85aa7c3ee6b55e59af9c27ded7319db19c19b2f5

What's the next step? Please tell me - I want to learn and contribute! ))))